dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -44,7 +44,9 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
+@Category(RemoteTests.class)
 public class DFSReadWriteTest
 {
     private static final String   clusterIP      = "192.168.56.101";
```

Comment:
this integration test was being run during maven install; added the correct category